### DISCUSSION TITLE
Updating Polyline's point in Active Selection

### DISCUSSION THREAD

**Discussion by umtdemr** [fabricjs/fabric.js#9101] [General]
Hi, I have a problem with polylines. What I want is, I want to change the first point of the polyline when a user moves the polyline with other objects so that the polyline can preserve its position. But it remains the same position since it has a group. How can I achieve this?

What I've tried is:

```js
const lineStartPoints = {
  x: 200,
  y: 150
}
canvas.on('object:moving', e => {
    const target = e.target;
    if (target.type === 'activeSelection') {
  	const objects = target.getObjects().filter(o => o.type === 'polyline');
   	for (const line of objects) {
    	    line.points[0].x = lineStartPoints.x;
            line.points[0].x = lineStartPoints.y;
            // these didn't help
            // line._setPositionDimensions({});
            // line.setCoords();
        }
    }
})
```

If you want to try online here is the [jsfiddle link](https://jsfiddle.net/mediumgoal/yjhregbk/33/). (Try selecting polyline and bottom rectangle and move them)

Thank you

**Comment by ShaMan123** [comment]
v6 is the only way forward with polyline and selection

**Reply by umtdemr** [reply]
@ShaMan123  Are you saying that if I switch to v6, this issue will be resolved?

**Reply by ShaMan123** [reply]
I am saying that is your only chance to make it work, I don't know though
